packages/synapse-react-client/src/components/DataGrid/SynapseGrid.tsx

```diff
@@ -372,111 +373,121 @@ const SynapseGrid = forwardRef<
             </div>
           )}
         </Grid>
-      </Grid>
-      {session && (
-        <>
-          {/* Grid Loading State */}
-          {!isGridReady && (
-            <>
-              <h3>Setting up grid...</h3>
-              <div style={{ marginBottom: '10px' }}>
-                {!session && <p>Creating grid session...</p>}
-                {session && !replicaId && <p>Setting up real-time sync...</p>}
-                {session && replicaId && !presignedUrl && (
-                  <p>Establishing secure connection...</p>
-                )}
-                {session && replicaId && presignedUrl && !isConnected && (
-                  <p>Connecting to server...</p>
-                )}
-                {isConnected && !isGridReady && <p>Loading table data...</p>}
-                <SkeletonTable numRows={4} numCols={1} />
-              </div>
-            </>
-          )}
 
-          {/* Grid */}
-          {isGridReady && (
-            <div>
-              <DataSheetGrid
-                value={rowValues}
-                columns={colValues}
-                rowKey="_rowId"
-                rowClassName={({ rowData, rowIndex }: any) =>
-                  classNames({
-                    'row-deleted': deletedRowIds.has(rowData._rowId),
-                    'row-created': createdRowIds.has(rowData._rowId),
-                    'row-updated': updatedRowIds.has(rowData._rowId),
-                    'row-valid': rowData.__validationResults?.isValid === true,
-                    'row-invalid':
-                      rowData.__validationResults?.isValid === false,
-                    'row-unknown':
-                      rowData.__validationResults?.isValid == undefined,
-                    'row-selected': selectedRowIndex === rowIndex,
-                  })
-                }
-                createRow={addRowToModel}
-                duplicateRow={({ rowData }: any) => ({
-                  ...rowData,
-                  _rowId: genId(),
-                })}
-                onChange={handleChange}
-                onActiveCellChange={({ cell }) => {
-                  setSelectedRowIndex(cell ? cell.row : null)
-                }}
-              />
-              {/* Show validation messages for selected row */}
-              {selectedRowIndex !== null &&
-                rowValues[selectedRowIndex] &&
-                Array.isArray(
+        {session && (
+          <>
+            {/* Grid Loading State */}
+            {!isGridReady && (
+              <Grid size={12}>
```

Comment:
I wrapped each of these sections in a flex grid, but did not change the contents